language-design.md

```diff
@@ -102,7 +289,43 @@ you do so? If not, why not?
 
 **Response**
 
-
+It may be possible to have a language be in the same "grain" as a natural 
+language without actually having the associated ambiguity. If it has a 
+well defined structure and syntax, then the flexibility described in the
+excerpt above may not be an issue.
+
+In Pavlus, it is described that "finding natural-language replacements for 
+some of the more abstruse syntax went a long way" [Pavlus 2012]. At first
+glance, this seems to be a direct contradiction to the experiment described
+in the above quote from Cook. However, in the case of Pavlus, novices were
+the ones being experimented on. For us, the more natural language-like 
+syntax in Pavlus was less easy to understand than the Java syntax. In Cook, 
+programmers were the ones using the natural language rather than novices.
+It would be interesting to perform an experiment about whether novices
+or experienced programmers have more or less trouble with a specific 
+natural language-like programming language. 
+
+Additionally, in Pavlus, there was more focus on the readability of the
+language rather than the writing - according to Pavlus, 
+> ```
+> integer i = 0
+> repeat 10 times
+> i = i + 1
+> end 
+> ```
+> That still looks mostly like Greek to me, but Stefik compares it to this equivalent 
+> statement in Java ("which is similar to Perl is some ways," he says):
+> ```
+> for(int i = 0; i < 10; i++) {
+> } 
+> ```
+> That’s not Greek, it’s Klingon 
+[Pavlus, 2012] 
+
+In both Pavlus and Cook, the code that resembled natural language
+was easier to read than the more standard programming language. This raises a 
+question about whether it is possible to strike a balance between that readability
+and the stricter structure apparently necessary. 
```

Comment:
This balance is extremely important.  My main concern with a natural language-based programming language like AppleScript is that if it's _too_ fluent, people may think that the vocabulary of the PL is equivalent to and seemingly limitless like a spoken language. That could be solved by a strict structure, but like you said, this is a difficult balance to find.